BooruSharp.Others/Pixiv.cs

```diff
@@ -106,6 +103,33 @@ private async Task UpdateTokenAsync()
         public override bool IsSafe()
             => false;
 
+        public override async Task AddFavoriteAsync(int postId)
+        {
+            var request = new HttpRequestMessage(new HttpMethod("POST"), _baseUrl + "/v2/illust/bookmark/add");
```

Comment:
```suggestion
            var request = new HttpRequestMessage(HttpMethod.Post, _baseUrl + "/v2/illust/bookmark/add");
```
[Microsoft](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpmethod?view=netstandard-2.0#remarks) recommends to use predefined HttpMethods for requests, but this suggestion is probably out of scope of this PR.

Reply:
Some others parts of the code were already using HttpMethod.Post so I changed it in Pixiv for consistency with 3a504542fd74a97ad2627c81faa0fa7e64189892